thirdparty/github.go

```diff
@@ -259,13 +259,17 @@ func GetCommitEvent(ctx context.Context, oauthToken, repoOwner, repo, githash st
 		return nil, APIResponseError{err.Error()}
 	}
 
-	grip.Debug(message.Fields{
+	msg := message.Fields{
 		"operation": "github api query",
 		"size":      resp.ContentLength,
 		"status":    resp.Status,
-		"commit":    githash,
+		"query":     githash,
 		"repo":      repoOwner + "/" + repo,
-	})
+	}
+	if commit != nil {
+		msg["commit"] = *commit.SHA
```

Comment:
maybe it would be better to also validate the SHA isn't nil?

Reply:
True. Fixed.